zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -154,7 +154,6 @@ public ZeppelinServer() {
 
   public static void main(String[] args) throws InterruptedException, IOException {
     ZeppelinServer.conf = ZeppelinConfiguration.create();
-    conf.setProperty("args", args);
```

Comment:
@zjffdu I think that this line has no effect. Is this correct?

Reply:
Yes，we could remove it